modules/unsupported/csv/src/test/java/org/geotools/data/csv/CSVWriteTest.java

```diff
@@ -307,8 +306,11 @@ public void appendContent() throws Exception {
         String contents2 = new String(baos2.toByteArray(), StandardCharsets.UTF_8);
 
         // Making sure whitespace doesn't cause problems
+        // and adjust for potential windows line endings.
         contents1 = contents1.replace(" ", "").trim();
         contents2 = contents2.replace(" ", "").trim();
+        contents1 = contents1.replace("\r", "");
+        contents2 = contents2.replace("\r", "");
```

Comment:
That sounds reasonable to me Jody.  I appreciate your help resolving this.  Thanks.